### PR TITLE
troubleshooting: fix positions.yaml location reference

### PR DESCRIPTION
**Pull request by gregwebs**

### CODE REVIEW

**Comment by daixiang0** [comment]
LGTM. good catch!

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ kubectl exec -it promtail-bth9q -- /bin/ash
 
 Once connected, verify the config in `/etc/promtail/promtail.yml` is what you expected
 
-Also check `/var/log/positions.yaml` and make sure promtail is tailing the logs you would expect
+Also check `/var/run/promtail/positions.yaml` and make sure promtail is tailing the logs you would expect
```
[the remark this answers is not in the document]
Comment (daixiang0) [reply docs/troubleshooting.md]: @gregwebs add a path about ksonnet is better.

[the remark this answers is not in the document]
Comment (gregwebs) [reply docs/troubleshooting.md]: okay, I will leave this fix in your hands so you can make it work for all the deployment methods.

Comment (Kuqd) [inline docs/troubleshooting.md]:
```suggestion
Also check `/var/log/positions.yaml` (default to `/var/run/promtail/positions.yaml` if you're using helm) and make sure promtail is tailing the logs you would expect
```

Comment (steven-sheehy) [reply docs/troubleshooting.md]: Again, the helm chart has it at `/run/promtail/positions.yaml`. But other than that sounds agreeable.

**Comment by daixiang0** [comment]
@cyriltovena please close it as fixed in 3370ccfd

**Pull request closed without merging**